bitcash/network/services.py

```diff
@@ -142,6 +142,14 @@ def get_transactions(cls, address):
             raise ConnectionError
         return r.json()['transactions']
 
+    @classmethod
+    def get_transactions_testnet(cls, address):
+        r = requests.get(cls.TEST_ADDRESS_API.format(address),
+                                            timeout=DEFAULT_TIMEOUT)
+        if r.status_code != 200:  # pragma: no cover
```

Comment:
I'm fine with this but what do you think about `r.raise_for_status()`? It can wait, no need to adjust especially if we are doing status code checking as it is.

Reply:
I did not know about this function - indeed it does exactly what we want! It would save us a number of lines throughout the code.

Reply:
Agreed, it's a handy one. Not crucial but good to know about.

Reply:
Updated! 😃